libs/ccc/coef/impl.py

```diff
@@ -640,7 +640,7 @@ def ccc(
 
     # get number of cores to use
     n_jobs = os.cpu_count() if n_jobs is None else n_jobs
-    default_n_threads = (os.cpu_count() - n_jobs) if n_jobs < 0 else n_jobs
+    default_n_threads = (os.cpu_count() + n_jobs) if n_jobs < 0 else n_jobs
```

Comment:
This fixes the problem described in the PR. However, if `n_jobs` has an absolute value equal or larger than `os.cpu_count()` it will be zero or negative, right? We should check if the final value for `default_n_threads` is within a valid range.

Reply:
And I think the docstring of this function needs to be updated, right?

Reply:
Yes that's what I missed! I'll update the code and add test cases. Thanks!